### PR TITLE
Refactor Scalar DB Server documentation

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

File: server/README.md [hunk 1]
```diff
@@ -1,49 +0,0 @@
-# Scalar DB Server
```
Comment (brfrn169) [inline server/README.md:1]: Moved the content of this README to `docs/scalardb-server.md`.

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

File: docs/scalardb-server.md [hunk 2]
```diff
@@ -17,27 +17,27 @@ Scalar DB server is written in Java. So the following software is required to ru
...
+$ ./gradlew installDist
 ```
 
 Of course, you can archive the jar and libraries by `./gradlew distZip` and so on.
 
 ## Configure Scalar DB Server
 
 You need a property file holding the configuration for Scalar DB Server. 
-It contains two sections, Server configurations and Underlining storage/database configurations.
+It contains two sections, Scalar DB Server configurations and underlining storage/database configurations.
```
Comment (feeblefakie) [inline docs/scalardb-server.md:33]:
```suggestion
It contains two sections: Scalar DB Server configurations and underlying storage/database configurations.
```

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!
Left suggestions on typos, so please merge after applying them.

**Pull request merged**